src/api/app/helpers/webui/user_activity_helper.rb

```diff
@@ -18,15 +18,15 @@ def contributions_percentiles(contributions_array)
 
   def activity_classname(activity, percentiles)
     if activity.zero?
-      'activity-percentil0'
+      'table-activity-percentil0'
```

Comment:
```suggestion
      ''
```

Mmm..I'd rather leave it empty or at least set the color definition to transparent if it is `zero`, no?

This is what I get at the moment
![image](https://github.com/user-attachments/assets/3c49b9c7-c815-4c19-87b8-38061c632058)


but this is how it really is behind
![image](https://github.com/user-attachments/assets/9cf073f8-34dc-40d8-9662-f1dc97e373a0)

Reply:
True. Way better. I used your suggestion and removed the `table-activity-percentil0` table variant. :+1: